datalake/src/main/java/com/sequenceiq/datalake/service/FreeipaService.java

```diff
@@ -36,12 +51,38 @@ public void checkFreeipaRunning(String envCrn) {
 
     }
 
-    DescribeFreeIpaResponse describe(String envCrn) {
+    public DescribeFreeIpaResponse describe(String envCrn) {
```

Comment:
nit: why changing this to `public`? Nothing calls it externally..

Reply:
Was needed for public prior to adding getNodeCount() which is instead public